eng/Version.Details.xml

```diff
@@ -105,7 +109,7 @@
       <Uri>https://github.com/dotnet/cli-migrate</Uri>
       <Sha>37e3a01aa1df320a286b0a858876f9994fb6bb18</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.CommandLineUtils" Version="1.0.1">
+    <Dependency Name="Microsoft.Extensions.CommandLineUtils" Version="3.0.0-preview5.19227.9">
```

Comment:
There is no such package. This was deprecated in 1.1. There won't be a new 3.0 version of this.